doc/UsersGuide/source/Configurations.rst

```diff
@@ -196,20 +196,136 @@ it will be expanded to cover the full range of ATM-only supported configurations
 
 Input files required for ATM configurations can be viewed in :numref:`Section %s <atm-in>`
 or in the `UFS WM RT Data Bucket <https://registry.opendata.aws/noaa-ufs-regtests/>`__. 
-Information on ``ufs.configure`` files is available in :numref:`Section %s <ufs-conf>`,
+Information on ``ufs.configure`` files is available in :numref:`Section %s <UFS-configurations>`,
```

Comment:
The ufs-conf section is specific to the ufs-conf section here, rather than the table of configurations: https://gsp-wm.readthedocs.io/en/us-27/InputsOutputs.html#ufs-configure-file.
```suggestion
Information on ``ufs.configure`` files is available in :numref:`Section %s <ufs-conf>`,
```